docs/source/release_notes/2021.1.rst

```diff
@@ -0,0 +1,73 @@
+
+pyMOR 2021.1 (September 30, 2021)
+--------------------------------
+We are proud to announce the release of pyMOR 2021.1! This release includes
+several new reductors for LTI systems. In particular, methods for reducing and
+analysing unstable systems have been added. ANNs can now be used in order to
+directly approximate output qunatities. Furthermore, it is now possible to
+work with time-dependent parameters in pyMOR.
+
+Over ??? single commits have entered this release. For a full list of changes
+see `here <https://github.com/pymor/pymor/compare/2020.2.x...2021.1.x>`__.
+
+pyMOR 2021.1 contains contributions by Tim Keil, Hendrik Kleikamp and Meret
+Behrens. See `here <https://github.com/pymor/pymor/blob/main/AUTHORS.md>`__ for
+more details.
+
+Release highlights
+^^^^^^^^^^^^^^^^^^
+
+Methods for unstable LTI systems
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Many popular system-theoretic model order reduction methods are not applicable
+to unstable LTI systems out of the box. In
+`[#1149] <https://github.com/pymor/pymor/pull/1149>`_ two reductors and several
+methods for working with and analyzing unstable |LTIModels| have been added.
+The :class:`~pymor.reductors.bt.FDBTReductor` allows for applying the balanced
+truncation technique to unstable systems by performing a Bernoulli stabiliztion
+before the using the classical BT method. The :class:`~pymor.reductors.h2.GapIRKAReductor`
+aims to compute a reduced order model such that the approximation error with
+respect to the :math:`\mathcal{H}_2`-Gap norm is small. Additionally,
+a variety of numerical linear algebra methods have been a part of
+`[#1149] <https://github.com/pymor/pymor/pull/1149>`_: Riccati equation solvers
+for small and dense matrices, Bernoulli matrix equation solver, new options for
+for pyMORs eigensolver such as shift-and-invert mode and :math:`\mathcal{L}_2`-norm
+computation for |LTIModels|. The new methods and reductors are showcased in
+:doc:`../tutorial_unstable_lti_systems`.
+
+Modal truncation
+~~~~~~~~~~~~~~~~
+
+Output reductor using ANNs
```

Comment:
I would not call that a "highlight" of the release. For me it is more like an additional new feature. If #1383 makes it into the release, that would be much more interesting I think.

Reply:
Agreed. We should change the release notes summary as soon as #1383 is merged.

Reply:
#1383 didn't make it, so no change here.